### PR TITLE
new design #66

### PR DESCRIPTION
**Pull request by RocktimSaikia**
Hi in this pull request I made few changes in the overall home page design .All the designs specs is based on the issue [#66](https://github.com/JasonFritsche/BreweryFinder/issues/66) that I opened few days ago.

So again these are the things i worked on :
-  Got rid of the heavy background image that was not optimized  instead of went for a more simple flat design with a simple vector png which made the loading time for the page more faster.
- Used some simple basic css design principles to make the web page look more professional and standard( like neon gradients ,flat design color palettes ,standard google fonts).
- Made the footer stick to bottom and inside the footer added a contribution button linking to this repo.

**here is the final result :**
brewer

Please let me know what you guys think of these chnages .

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,59 +1,99 @@
-@import url("https://fonts.googleapis.com/css?family=Kreon|Quicksand");
+@import url('https://fonts.googleapis.com/css?family=Kreon|Quicksand|Pacifico|Lobster|Raleway:600,800');
```
Comment (cdrani) [inline src/App.css]: Font loading takes some time to load, especially fonts that aren't native to the browser. Are these new fonts necessary/integral to the design? If so, then the old ones should be removed. I recommend only 2-3 fonts.

File: src/App.css [hunk 2]
```diff
@@ -1,59 +1,99 @@
...
   /* Full height */
   height: 100%;
 
   /* Center and scale the image nicely */
-  background-position: center;
+  /* background-position: center;
   background-repeat: no-repeat;
-  background-size: cover;
+  background-size: cover; */
 }
```
Comment (cdrani) [inline src/App.css]: There's already a body selector in `src/index.css`. Perhaps move the above css into `src/index.css`?

File: src/App.css [hunk 3]
```diff
@@ -1,59 +1,99 @@
...
-.Glow {
+/* .Glow {
   color: #61dafb;
   font-size: 4.5em;
   color: #00b6ff;
   -webkit-animation: glow 1s ease-in-out infinite alternate;
   -moz-animation: glow 1s ease-in-out infinite alternate;
   animation: glow 1s ease-in-out infinite alternate;
-}
+} */
```
Comment (cdrani) [inline src/App.css]: Please prefer classes over ids to reduce specificity as it's unneeded here. Could you also add a new line between each selector to space them out? Additionally remove the commented out css.

File: src/components/BrewerySearch.js [hunk 4]
```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types'
 import Typed from 'react-typed'
 import * as Alert from 'react-bootstrap/Alert'
 import RadioButtonContainer from './RadioButtonContainer'
+const img = require('../public/cheers.png')
+const githublogo = require('../public/github-logo.png')
```
Comment (cdrani) [inline src/components/BrewerySearch.js]: Please use esmodule `imports` over commonjs `require` for uniformity.

**Comment by RocktimSaikia** [comment]
@cdrani Thanks for the review i will make your requested changes asap 😊

**Comment by RocktimSaikia** [comment]
@cdrani hi i made the requested changes that you mentioned. So again here is the overview of the new changes i made :
  - Removed the unused old google fonts that were taking time to load.
  - Moved the body selector css part to ```index.css``` file  .
  - Used classes over ids and added newline between every css selector
  - Replaced commonjs require with es6 modules.
  - Fixed the gradient background issue that was making the ``` go back button``` hard to read. 

A few new changes i made : 
  - Changed the background color of search input field from gradient to plain white.Which looks more clean now. 
  - Added gradient background to cards and made the card buttons block level which looks much better now.

Here is the changed homepage with the white colored input field  :
new site

Search result page with new gradient cards : 
bak

**Comment by JasonFritsche** [comment]
@RocktimSaikia Thanks for your contributions to the project. This is an awesome addition.

**Comment by RocktimSaikia** [comment]
I am glad i could use this holiday season working on open source rather than my college stuff :)

**Comment by JasonFritsche** [comment]
Feel free to make more PR's. If you see a bug or feature that is not in the current set of issues then feel free to open a new issue. Thanks!

**Pull request merged**